### ISSUE TITLE
SyntaxError: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by khelle** [polkadot-js/api#3735]
Hello,

when I try to add @polkadot-js/api inside my project I receive following error:

```
SyntaxError: Cannot use import statement outside a module
```

It errors on simple code as:
```
const Polka = require('@polkadot/api');
```

My node version is 12, but I did no see any requirements considering node version in repo or docs and read in https://github.com/polkadot-js/api/issues/3430 that "its supposed to work on node 12".

The question then is - if it is supposed to work on node 12 - how can I make it work?

EDIT:

I tried using

```
import { ApiPromise,WsProvider } from "@polkadot/api";
```

on node 14 and have exactly the same problem - "SyntaxError: Cannot use import statement outside a module"

**Comment by jacogr** [comment]
It should not install on node < 14, see https://github.com/polkadot-js/api/blob/master/packages/api/package.json#L12 (older version missed that field). 

Technically, yes, it should work, in practice the package required Node 14.

**Comment by khelle** [comment]
@jacogr I just tried on node 14: (14.17 to be precise)

```
import { ApiPromise,WsProvider } from "@polkadot/api";
```

and have exactly the same error. "SyntaxError: Cannot use import statement outside a module"

**Comment by jacogr** [comment]
Read about Node and using import vs require. There is a difference between CJS (Node default) and ESM packages.

import is only applicable in ESM, which means one of -

(a) you have `"type": "module" in your package.json
(b) you call the import from a  file with an mjs extension

Require is only applicable in CJG, which means one of -

(a) you have no type specifier in your package.json
(b) you have a commonjs specified in your package.json for the type specifier
(c) you call it from file with cjs extension

TL:DR When you do nothing using Node, cjs is always the default.
